src/providers/lock-screen/lock-screen.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable } from '@angular/core';
+import { Platform } from 'ionic-angular';
+
+declare let cordova: any;
+
+@Injectable()
+export class LockScreenProvider {
```

Comment:
"Lock screen" isn't the clearest name for this I think. We have numerous types of screen "locking", e.g. orientation, prevent sleep, guided access etc. Could we change it to mirror what the cordova plugin was called? So `DeviceAuthenticationProvider`?